articles/hardware/hs64/ts4231.md

```diff
@@ -3,10 +3,11 @@ uid: hs64_ts4231
 title: Headstage64 Position Data
 ---
 
-The following excerpt from the Headstage64 [example workflow](xref:hs64_hs64) demonstrates electrical stimulation.
+The following excerpt from the Headstage64 [example workflow](xref:hs64_hs64) demonstrates the headstage's position tracking capabilities.
 
 ::: workflow
 ![/workflows/hardware/hs64/ts4231.bonsai workflow](../../../workflows/hardware/hs64/ts4231.bonsai)
 :::
 
-The <xref:OpenEphys.Onix1.TS4231V1PositionData> operator generates a sequence of <xref:OpenEphys.Onix1.TS4231V1PositionDataFrame>s.
\ No newline at end of file
+The <xref:OpenEphys.Onix1.TS4231V1PositionData> operator generates a sequence of [TS4231V1PositionDataFrames](xref:OpenEphys.Onix1.TS4231V1PositionDataFrame). `TS4231V1PositionData` emits a `TS4231V1PositionDataFrame` when a receiver on the headstage successfully receives a coordinated set of four IR sweeps - two from each lighthouse. These coordinated sweeps occur at a rate of 30 Hz, so the maximum achievable sample rate is 30 Hz. In practice, the sample rate depends on the conditions of the behavioral arena. Achieving a 30 Hz sample rate requires that that the receiver is within range of the lighthouses and that there is no occlusion between the receiver and the lighthouses. There are four receivers on the headstage64 to increase the likelihood that at least one of them successfully receiving a complete lighthouse signal. Each receiver that successfully receives the full lighthouse signal results in `TS4231V1PositionData` emitting a `TS4231V1PositionDataFrame`. This means it is possible for the `TS4231V1PositionData` operator to emit multiple `TS4231V1PositionDataFrames` for a given set of coordinated sweeps.
```

Comment:
IR: Infrared
sweeps: what are they? why are they relevant? If you are going to provide this level of detail, you need to setup what you are talking about.

Diagram would be good with call outs.

Reply:
I think you're right. I'm going to limit this description to as brief of an overview as I think is necessary. Instead, we should link out to the hardware docs if users want those details. If my latest commit looks good to you, please resolve conversation